include/eve/memory/pointer.hpp

```diff
@@ -20,10 +20,10 @@ namespace eve::detail
     else                                   return p;
   }
 
-  template <typename U, typename T, std::size_t size>
-  EVE_FORCEINLINE auto ptr_cast(eve::aligned_ptr<T, size> p)
+  template <typename U, typename T, typename Lanes>
+  EVE_FORCEINLINE auto ptr_cast(eve::aligned_ptr<T, Lanes> p)
   {
-    return aligned_ptr<U, size>{(U*)(p.get())};
+    return aligned_ptr<U, Lanes>{(U*)(p.get())};
```

Comment:
Not `Lanes`. Alignment in bytes should stay the same.

So `eve::fixed<Lanes() * sizeof(T) / sizeof(U)>`